local.settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+  }
+}
```

Comment:
Do you have a `.gitignore` file? Add the name if this file to there so that it isn't versioned. :)